Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst

```diff
@@ -57,17 +57,17 @@ Setup and Start DDS Daemon
 
 .. note::
 
-	This step needs to be performed every time the user nublado instance is restarted, or if the user shuts down the daemon.
+	This step needs to be performed every time the user Nublado instance is restarted, or if the user shuts down the daemon.
 
 In the terminal enter the following sequence of commands to setup your environment;
 
 .. prompt:: bash
 
   source ${LOADSTACK}
-  export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
   source $OSPL_HOME/release.com
   export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
 
+
 Before starting the daemon, make sure to check if it is already running or not;
 
 .. code-block:: shell-session
```

Comment:
There is one more 

```
export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
```

in line 122 that can be removed as well.. (I can't comment on it.. :( )

Reply:
I think the only mention left of OSPL_HOME being defined (in the repos I have checked out) is in ts_cycle_build ... but presumably that has to be there :wink: